.github/workflows/lint.yml

```diff
@@ -100,3 +100,11 @@ jobs:
           opam exec -- pre-commit run --verbose --all lint-ocaml
           # to debug errors in pre-commit, use instead:
           # opam exec -- pre-commit run --verbose --all lint-ocaml || cat /root/.cache/pre-commit/pre-commit.log
+
+  github-actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: reviewdog/action-actionlint@v1
```

Comment:
Nice! Now we need to port those actionlint checks in Semgrep!

Reply:
Maybe you can remove the TODO? comment now next to the name: lint at the top.

Reply:
Thanks for the suggestion, I'll consider this for a follow up PR.